.travis.yml

```diff
@@ -22,7 +20,8 @@ before_script:
   - echo "DEBIAN_FRONTEND=noninteractive" >> travis.env
   - echo "HAS_JOSH_K_SEAL_OF_APPROVAL" >> travis.env
   - echo $SPARK_VERSION >> travis.env
+  - ./gradlew startTestEnv
 
 script:
-  # run test container
-  - docker run --env-file travis.env -v $(pwd)/src/test/resources/log4j.properties:/usr/local/spark/conf/log4j.properties -v $(pwd):/opt/salt --rm --workdir="/opt/salt" uncharted/sparklet:$SPARK_VERSION ./gradlew coverage coveralls
+  # build and testcd
```

Comment:
Haha, I copied and pasted directly from your change. Then I noticed that it said cd. And then I thought, maybe it stood for container or docker or something? So i left it as is. Live and Learn.